juniper-vpn.py

```diff
@@ -1,17 +1,16 @@
-#!/usr/bin/env python2.7
-# -*- coding: utf-8 -*-
+#!/usr/bin/env python3
```

Comment:
This commit LGTM

Reply:
Yep, that's the blanket 2to3, so its easy to re-roll.